includes/pdf_upload.form.inc

```diff
@@ -18,6 +18,33 @@
  */
 function islandora_scholar_pdf_upload_form(array $form, array &$form_state) {
   $upload_size = min((int) ini_get('post_max_size'), (int) ini_get('upload_max_filesize'));
+
+  // Pull document versions from the module settings if available.
+  if (variable_get('islandora_scholar_specify_document_versions', FALSE)) {
+    $document_version_names = array_map('trim', explode("\n", variable_get('islandora_scholar_document_versions', "Author's Original\nSubmitted Manuscript Under Review\nAccepted Manuscript\nProof\nVersion of Record\nCorrected Version of Record\nEnhanced Version of Record")));
+    $document_versions = array_combine($document_version_names, $document_version_names);
```

Comment:
Could get into trouble with translations here... Drupal variables can be translated... But the way it is done could lead to different languages having different sets of terms... there's no strong way to correlate between them.

Same below, with the `$use_permissions`.